Doc/library/faulthandler.rst

```diff
@@ -170,3 +173,8 @@ handler:
       File "<stdin>", line 1 in <module>
     Segmentation fault
 
+
+.. seealso::
```

Comment:
Would it be possible to put it at the top, rather than at the end? Same remark for pdb.rst.